lib/statsd/instrument.rb

```diff
@@ -344,9 +344,9 @@ def statsd_remove_distribution(method, name)
       remove_from_method(method, name, :distribution)
     end
 
-    VoidClass = Class.new(BasicObject)
+    VoidClass = Class.new
     private_constant :VoidClass
-    VOID = VoidClass.new
+    VOID = VoidClass.new.freeze
```

Comment:
Will freezing this object does anything? I don't think it has a state, so there is nothing you can do to change that state.

Reply:
Not really, I did this more as "best practice" for singleton objects that should not change